test/cbmc/negative_tests/README.md

```diff
@@ -0,0 +1 @@
+../aws-templates-for-cbmc-proofs/template-for-repository/negative_tests/README.md
```

Comment:
What is the purpose of the `test/cbmc/negative_tests` directory?

Reply:
Right now the CBMC proof does not require negative tests. I have removed the directory.